backend/webdav/webdav.go

```diff
@@ -458,7 +465,17 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 		precision:   fs.ModTimeNotSupported,
 	}
 
-	client := fshttp.NewClient(ctx)
+	var client *http.Client
```

Comment:
Move this code to `fs/fshttp` as `NewClientWithUnixSocket` maybe and call it here `fshttp.NewClientWithUnixSocket(ctx, opt.UnixSocket)`

Reply:
Done